Templates.xcodeproj/project.pbxproj

```diff
@@ -216,7 +216,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [[ ! $CI ]]; then\n    rake lint:tests\n    rake lint:output\nfi";
+			shellScript = "if [[ ! $CI ]]; then\n  [[ -r ~/.profile ]] && source ~/.profile\n  [[ `which rbenv` ]] && eval \"$(rbenv init - --no-rehash sh)\"\n  if [[ `which bundler` ]]; then\n    bundle exec rake lint:tests\n    bundle exec rake lint:output\n  else\n    echo \"warning: bundler not found, can't lint code. Please install bundler then run bundle install.\"\n  fi\nfi";
```

Comment:
I get errors with the bundler stuff. I'm telling you, my ruby stuff is always messed up, it's cursed 😢 
I'll add that rbenv in my .profile, hopefully nothing else breaks 🤞